tests/test_cifar_format.py

```diff
@@ -125,6 +125,56 @@ def test_can_save_and_load_empty_image(self):
             compare_datasets(self, dataset, parsed_dataset,
                 require_images=True)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_and_load_cifar100(self):
+        source_dataset = Dataset.from_iterable([
+            DatasetItem(id='image_2', subset='test',
+                image=np.ones((32, 32, 3)),
+                annotations=[Label(0, attributes={
+                    'coarse_label': 0
+                })]
+            ),
+            DatasetItem(id='image_3', subset='test',
+                image=np.ones((32, 32, 3))
+            ),
+            DatasetItem(id='image_4', subset='test',
+                image=np.ones((32, 32, 3)),
+                annotations=[Label(1, attributes={
+                    'coarse_label': 0
+                })]
+            )
+        ], categories=[['label_0', 'superclass_0'], ['label_1', 'superclass_0']])
+
+        with TestDir() as test_dir:
+            CifarConverter.convert(source_dataset, test_dir, save_images=True)
```

Comment:
How is it different from CIfar10 tests? Can we choose between formats?

Reply:
In this test, the label has a parent and each label has a `coarse_label` attribute. No, we cannot choose the format. But if the annotation file contains fine labels and coarse labels, then it will be cifar-100. If this is not the case, then there is no point in using this format.

Reply:
I feel, we need to make `coarse/fine` information automatically inferred from `categories` and add an option to converter to use 10 or 100 format.